core/plugins/org.fusesource.ide.camel.model.service.impl.v2232fuse790054redhat00001/.classpath

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8">
+		<attributes>
+			<attribute name="maven.pomderived" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry exported="true" kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="src/"/>
+	<classpathentry exported="true" kind="lib" path="libs/camel-blueprint-2.23.2.fuse-790054-redhat-00001.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/camel-catalog-2.23.2.fuse-790054-redhat-00001.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/camel-catalog-maven-2.23.2.fuse-790054-redhat-00001.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/camel-catalog-provider-karaf-2.23.2.fuse-790054-redhat-00001.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/camel-catalog-provider-springboot-2.23.2.fuse-790054-redhat-00001.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/camel-core-2.23.2.fuse-790054-redhat-00001.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/camel-spring-2.23.2.fuse-790054-redhat-00001.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/log4j-over-slf4j-1.7.25.redhat-00001.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/slf4j-api-1.7.25.redhat-00001.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/slf4j-simple-1.7.25.redhat-00001.jar"/>
```

Comment:
to check that it is not a newer version used by Fuse https://maven.repository.redhat.com/ga/org/slf4j/log4j-over-slf4j/1.7.30.redhat-00001/ (available on Maven Red hat repository GA)

Reply:
this is still 1.7.25 https://github.com/jboss-fuse/camel/blob/5a87873acc1705084c99735f741fcfc31a26c81f/parent/pom.xml#L652